client/signup/steps/theme-selection/index.jsx

```diff
@@ -58,8 +59,7 @@ class ThemeSelectionStep extends Component {
 			<SignupThemesList
 				surveyQuestion={ this.props.chosenSurveyVertical }
 				designType={ this.props.designType || this.props.signupDependencies.designType }
-				handleScreenshotClick={ this.pickTheme }
-			/>
```

Comment:
I've seen this closing part of the tag written on a newline as well as at the end of the previous line in Calypso. Is there any mention in the coding standards for which is preferred? I haven't seen anything.